frontend/app/calculators/transportation/trip-results.js

```diff
@@ -24,6 +24,8 @@ export default class TransportationTripResultsCalculator extends EmberObject {
   }
 
   // Returns an object whose keys are temporal types
+  // TODO: It looks like many of the EmberObject parameters (documented above on lines 7-17)
+  // are used in this CP but not added to the computed signature. Add them to this signature.
```

Comment:
@godfreyyeung thanks for noticing and flagging this as a future todo

Reply:
One thing I notice looking at this file is that it's just very long, I didn't read through to see exactly what's happening here, as it seems complex (maybe just my lack of Ember knowledge), but I wonder if there are multiple things happening that may be broken into helper functions that could be imported to be more declarative / easier to follow.